File: AccuPay/Domain/TimeLogs/BreakTimeBracket.vb
Option Strict On

Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Global.AccuPay.Entity

    <Table("breaktimebracket")>
    Public Class BreakTimeBracket

        <Key>
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property RowID As Integer

        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property Created As Date

        Public Property CreatedBy As Integer?

        <DatabaseGenerated(DatabaseGeneratedOption.Computed)>
        Public Property LastUpd As Date?

        Public Property LastUpdBy As Integer?

        Public Property ShiftDuration As Decimal

        Public Property BreakDuration As Decimal

        Public Property DivisionID As Integer

        <ForeignKey("DivisionID")>
        Public Overridable Property Division As Division

    End Class

End Namespace

File: AccuPay.Core/Entities/BreakTimeBracket.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Core.Entities
{
    [Table("breaktimebracket")]
    public class BreakTimeBracket
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int RowID { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public DateTime Created { get; set; }

        public int? CreatedBy { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public DateTime? LastUpd { get; set; }

        public int? LastUpdBy { get; set; }

        public decimal ShiftDuration { get; set; }

        public decimal BreakDuration { get; set; }

        public int DivisionID { get; set; }

        [ForeignKey("DivisionID")]
        public virtual Division Division { get; set; }
    }
}